src/ui/qgsrendererv2propsdialogbase.ui

```diff
@@ -102,7 +102,7 @@
             <bool>false</bool>
            </property>
            <property name="collapsed" stdset="0">
-            <bool>false</bool>
+            <bool>true</bool>
```

Comment:
@nyalldawson , done -- I swear I had tried it before and found it not to work, but turns out it must have all been in my head ;) commit updated, will merge when green.